rasa/core/policies/ted_policy.py

```diff
@@ -323,9 +327,13 @@ def __init__(
         """Declare instance variables with default values."""
         self.split_entities_config = rasa.utils.train_utils.init_split_entities(
             kwargs.get(SPLIT_ENTITIES_BY_COMMA, SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE),
-            self.defaults[SPLIT_ENTITIES_BY_COMMA],
+            self.defaults.get(
+                SPLIT_ENTITIES_BY_COMMA, SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE
+            ),
         )
 
+        # TODO: check if the else statement can be removed.
+        #  More context here - https://github.com/RasaHQ/rasa/issues/5786#issuecomment-840762751
```

Comment:
It's a Todo from other part of the codebase and not necessarily for this feature. Once enable has closed this issue, we should refactor the code here.
I think it makes sense to request enable to let us know once the issue is closed.

Reply:
I think we should create a separate issue for the TODO, but place it in the blocked column. It will be revisited as part of the refinement meeting then.